knojector/steam-authentication-bundle/1.0/config/packages/knojector_steam_authentication.yaml

```diff
@@ -0,0 +1,5 @@
+knojector_steam_authentication:
```

Comment:
Maybe a link to your documentation so I know where to go if I need support/docs?